apikey_scope_editable/wizards/res_users_apikeys_description.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+from odoo.addons.base.models.res_users import check_identity
+
+
+class APIKeyDescription(models.TransientModel):
+    _inherit = "res.users.apikeys.description"
+
+    scope = fields.Char()
```

Comment:
Shouldn't this be a selection to avoid errors, or at least to have a hint about what to fill?